### PR TITLE
[STY] apply black to surface subpackage

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Closes none but relates to #3861

### CODE REVIEW

File: nilearn/surface/surface.py [hunk 1]
```diff
@@ -157,26 +172,33 @@ def _ball_sample_locations(
 
     """
     if depth is not None:
-        raise ValueError("The 'ball' sampling strategy does not support "
-                         "the 'depth' parameter")
+        raise ValueError(
+            "The 'ball' sampling strategy does not support "
+            "the 'depth' parameter"
```
Comment (bthirion) [inline nilearn/surface/surface.py:177]: The error message is not great. Maybe we could take this opportunity to improve it...

**Review by bthirion — COMMENTED** [review]
LGTM otherwise.

File: nilearn/surface/surface.py (hunk 1 above)
Comment (Remi-Gau) [reply nilearn/surface/surface.py:177]: will try to improve it

Comment (Remi-Gau) [reply nilearn/surface/surface.py]: I at least added a tip on how to avoid the error.

IMO this error should just be a warning.

**Pull request merged**